tests/test_plugin_loader_edge_cases.py

```
@@ -663,6 +667,114 @@ def test_get_plugin_dirs_missing_local_permission_path_is_skipped(self):
+    def test_should_ignore_plugin_file_helper(self):
+        from mmrelay.plugin_loader import _should_ignore_plugin_file
+
+        for pattern in PLUGIN_IGNORED_FILE_PATTERNS:
+            if pattern.startswith("test_"):
+                assert _should_ignore_plugin_file(
+                    pattern.replace("*", "foo")
+                ), f"Should ignore {pattern.replace('*', 'foo')}"
+            elif pattern.endswith("_test.py"):
+                assert _should_ignore_plugin_file(
+                    pattern.replace("*", "foo")
+                ), f"Should ignore {pattern.replace('*', 'foo')}"
```

Ruff (0.15.12)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)